internal/controller/envoy_gateway_auth_cluster_reconciler.go

```diff
@@ -190,7 +190,7 @@ func (r *EnvoyGatewayAuthClusterReconciler) buildDesiredEnvoyPatchPolicy(authori
 	}
 
 	authorinoServiceInfo := authorinoServiceInfoFromAuthorino(authorino)
-	jsonPatches, err := kuadrantenvoygateway.BuildEnvoyPatchPolicyClusterPatch(kuadrant.KuadrantAuthClusterName, authorinoServiceInfo.Host, int(authorinoServiceInfo.Port), authClusterPatch)
+	jsonPatches, err := kuadrantenvoygateway.BuildEnvoyPatchPolicyClusterPatch(kuadrant.KuadrantAuthClusterName, authorinoServiceInfo.Host, int(authorinoServiceInfo.Port), false, authClusterPatch)
```

Comment:
assuming mtls param is always false here as we can't do mtls with envoy gateway?

Reply:
right

> we can't do mtls with envoy gateway
I want to validate that statement myself. I will investigate in following up work. Not part of this PR, though.